common/analysis/command_file_lexer.cc

```diff
@@ -74,4 +75,13 @@ std::vector<TokenRange> CommandFileLexer::GetCommandsTokenRanges() {
   return commands;
 }
 
+bool CommandFileLexer::KeepSyntaxTreeTokens(const verible::TokenInfo& t) {
+  switch (t.token_enum()) {
+    case CFG_TK_NEWLINE:
```

Comment:
don't we intend to remove comment tokens before parsing?

or if this function is only used for the sake of testing (not needed publicly in non-tests), you could define it as a free function only in the test.  (It looks like you handle comments separately in the parsing state machine)